File: fix_imports.py
import os
import re

root_dir = r"c:\Users\VIKASH\msteamMAGIC\src"
patterns = [
    (re.compile(r"['\"]\.\.\/.*lib/prisma['\"]"), '"@/lib/prisma"'),
    (re.compile(r"['\"]\.\.\/.*lib/colorUtils['\"]"), '"@/lib/colorUtils"')
]

for root, dirs, files in os.walk(root_dir):
    for file in files:
        if file.endswith((".ts", ".tsx")):
            path = os.path.join(root, file)
            with open(path, 'r', encoding='utf-8') as f:
                content = f.read()
            
            new_content = content
            for pattern, replacement in patterns:
                new_content = pattern.sub(replacement, new_content)
            
            if new_content != content:
                print(f"Updating {path}")
                with open(path, 'w', encoding='utf-8') as f:
                    f.write(new_content)

File: fix_imports.js
const fs = require('fs');
const path = require('path');

const rootDir = 'c:\\Users\\VIKASH\\msteamMAGIC\\src';
const patterns = [
    { regex: /['"]\.\.\/.*lib\/prisma['"]/g, replacement: '"@/lib/prisma"' },
    { regex: /['"]\.\.\/.*lib\/colorUtils['"]/g, replacement: '"@/lib/colorUtils"' }
];

function walk(dir) {
    const files = fs.readdirSync(dir);
    files.forEach(file => {
        const filePath = path.join(dir, file);
        const stat = fs.statSync(filePath);
        if (stat.isDirectory()) {
            walk(filePath);
        } else if (file.endsWith('.ts') || file.endsWith('.tsx')) {
            let content = fs.readFileSync(filePath, 'utf8');
            let newContent = content;
            patterns.forEach(p => {
                newContent = newContent.replace(p.regex, p.replacement);
            });
            if (newContent !== content) {
                console.log(`Updating ${filePath}`);
                fs.writeFileSync(filePath, newContent, 'utf8');
            }
        }
    });
}

walk(rootDir);
